src/config.cpp

```diff
@@ -13,12 +13,19 @@
 // limitations under the License.
 
 #include "config.hpp"
+#include "log.hpp"
 
 #include <cassert>
+#include <filesystem>
+#include <fstream>
+#include <iostream>
+#include <stdlib.h>
+#include <unordered_map>
+#include <vector>
 
 const config_struct config;
 
-namespace {
+namespace configs {
```

Comment:
made a new name space for testing purposes. If this is okay I will file another issue to do more test coverage for these funcs.

Reply:
If you switch to using `unit.hpp` instead of `config.hpp` in the test, it should not be needed I guess

Reply:
looks like I cant still access the anon namespace funcs outside of the file. Lets keep it i guess?